tizen/src/extractor.h

```diff
@@ -0,0 +1,145 @@
+#ifndef FLUTTER_PLUGIN_WORKMANAGER_EXTRACTOR_H_
+#define FLUTTER_PLUGIN_WORKMANAGER_EXTRACTOR_H_
+
+#include <flutter/encodable_value.h>
+#include <flutter/method_call.h>
+
+#include <string>
+
+#include "constants.h"
+#include "options.h"
+#include "tasks.h"
+#include "utils.h"
+
+typedef flutter::MethodCall<flutter::EncodableValue> FlMethodCall;
+
+ExistingWorkPolicy ExtractExistingWorkPolicyFromCall(const FlMethodCall &call) {
+    std::string value;
+    const auto &args = *call.arguments();
+    flutter::EncodableMap map = std::get<flutter::EncodableMap>(args);
+    bool succeed = GetValueFromEncodableMap<std::string>(
+        &map, constants::keys::kExistingWorkpolicykey, value);
+    if (succeed) {
+        Upper(value);
```

Comment:
Should we use `Upper`?

Reply:
In android plugin they use `uppercase()` so i used `Upper`. [uppercase used in android plugin](https://github.com/fluttercommunity/flutter_workmanager/blob/44ba91f2ab527688bd2c4fab270caaa2766ec707/android/src/main/kotlin/be/tramckrijte/workmanager/Extractor.kt#L240C1-L240C1) But if we know the exact value passed from dart, this they would not be necessary.

Reply:
removed it